### ISSUE TITLE
DB/Quest: Update quest_objectives and quest_visual_effect

### ISSUE THREAD

**Issue by osleyder85** [TrinityCore/TrinityCore#26983]
### Description

- Update quest_objectives and quest_visual_effect for quests:
   * 8325 Reclaiming Sunstrider Isle
   * 8326 Unfortunate Measures
   * 8334 Agression
   * 37439 Thirst Unending
   * 37442 The Shrine of Dath'Remar
   * 37440 A Fistful of Slivers

### SQL fix

```sql
UPDATE `quest_objectives` SET `VerifiedBuild`=40120 WHERE `ID` IN (256445,256853,256854,259624,276209,276210,276219,276221);

UPDATE `quest_visual_effect` SET `VerifiedBuild`=40120 WHERE `ID` IN (259624,276219,276221) AND `Index`=0;
```

### Branch

master

### TC rev. hash/commit

TrinityCore rev. 0c6d05c 2021-09-29 23:36:00 +0200

**Comment by mdX7** [comment]
i think we dont require `VerifiedBuild` only updates, updating it while fixing something (like you do in the other issues) is good though! :)

**Comment by offl** [comment]
Imo you can simply suggest nearly everything parser suggests without spending time on splitting it into separate issues
For example those fixes are all related to quests, #26981

**Comment by osleyder85** [comment]
> 
> 
> Imo you can simply suggest nearly everything parser suggests without spending time on splitting it into separate issues For example those fixes are all related to quests, #26981

Those quests have already been reviewed and this is missing data.
